src/ExceptionList.cc

```
@@ -0,0 +1,106 @@
+ExceptionList::ExceptionList(const InternalSettingsList &exceptions)
+    : m_exceptions(exceptions)
+{
+}
+
+void ExceptionList::readConfig(KSharedConfig::Ptr config)
+{
+    m_exceptions.clear();
+
+    QString groupName;
+    for (int index = 0; config->hasGroup(groupName = exceptionGroupName(index)); ++index) {
+        InternalSettingsPtr configuration(new InternalSettings());
```

Clang (14.0.6)

[warning] variable 'configuration' is not initialized

(cppcoreguidelines-init-variables)